web/src/title/loader.ts

```diff
@@ -29,6 +34,11 @@ export async function loadProject(projectID: ProjectID) {
     let { osmBuffer, demandBuffer, contextDataBuffer, boundary } =
       await getInputFiles(project);
     console.timeEnd("get input files");
+    loadingMessage.set("Download finished, setting up project");
+    // TODO The animation won't work, because we block the UI thread below
```

Comment:
When we do any CPU-bound step, it blocks the UI thread, so the browser's progress animation stops. Eventually we might want to make everything run async in web workers, but not important for now